src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php

```diff
@@ -70,18 +73,23 @@ public function indexAction(AuthorizedApplicationsFilters $filters): Response
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller')) || is_granted('update', request.get('_legacy_controller')) || is_granted('delete', request.get('_legacy_controller')) || is_granted('read', request.get('_legacy_controller'))")
+     */
     public function viewAction(): void
     {
         // TODO: Implement viewAction() method in view PR.
         throw new NotImplementedException();
     }
 
     /**
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
+     *
      * @param Request $request
      *
      * @return Response
      */
-    public function createAction(Request $request): Response
+    public function createAction(): Response
```

Comment:
```suggestion
    public function createAction(Request $request): Response
```

Yous should favor modifications from develop branch, this is breaking the controller